controllers/disruption_controller.go

```diff
@@ -424,6 +436,12 @@ func (r *DisruptionReconciler) startInjection(instance *chaosv1beta1.Disruption)
 				continue
 			}
 
+			if disKind == chaostypes.DisruptionKindNodeFailure && instance.Spec.StaticTargeting && injection.InjectionStatus != chaostypes.DisruptionInjectionStatusNotInjected {
```

Comment:
Does this part of the code is coverage by unit test?

Reply:
It is now